src/hooks/useMediaQuery/useMediaQuery.stories.mdx

```diff
@@ -4,38 +4,71 @@ import { Meta } from '@storybook/blocks';
 
 # useMediaQuery
 
-Hook that returns a boolean indicating whether the media query matches.
+Hook that returns true when the media query matches.
 
-## Augmenting the MediaQuery type
+## Reference
+
+```ts
+function useMediaQuery(mediaQueryOrVariableName: string, defaultValue = false): boolean;
+```
 
-The `MediaQuery` type is used for the `mediaQueryVariableName` parameter of `useMediaQuery`.
+## Using the hook with a query string
 
-Augment the `MediaQuery` interface to add the names for the CSS variables that describe media
-queries in your project.
+The `useMediaQuery` hook can be used with a query string.
 
 ```tsx
-declare module '@mediamonks/react-hooks' {
-  interface MediaQuery {
-    '--min-width-480': never;
-    '--min-width-820': never;
-    '--min-width-768': never;
-    '--min-width-1024': never;
-  }
+function MyComponent(): ReactElement {
+  const isMinWidth480px = useMediaQuery('(min-width: 480px)');
+
+  return <div>{isMinWidth480px ? 'large' : 'small'}</div>;
 }
 ```
 
-## Reference
+## Using the hook with CSS variables
 
-```ts
-function useMediaQuery(mediaQueryVariableName: keyof MediaQuery, defaultValue = false): boolean;
+Define the media query in your CSS variables.
+
+```css
+:root {
+  --min-width-480: (min-width: 480px);
+  --min-width-820: (min-width: 820px);
+  --min-width-768: (min-width: 768px);
+  --min-width-1024: (min-width: 1024px);
+}
 ```
 
-## Usage
+Use the hook in your component using the CSS variable using the variable name as the first
+parameter.
 
 ```tsx
-function DemoComponent() {
-  const isMinWidth428px = useMediaQuery('--min-width-428', true);
+function MyComponent(): ReactElement {
+  const isMinWidth820px = useMediaQuery('--min-width-820');
 
-  return <div></div>;
+  return <div>{isMinWidth820px ? 'large' : 'small'}</div>;
 }
 ```
+
+### Augmenting the MediaQueryVariables interface
+
+The `MediaQueryVariables` type is used in the type for the `mediaQueryOrVariableName` parameter of
+`useMediaQuery`.
+
+Augment the `MediaQueryVariables` interface to enable auto-complete suggestions for the CSS
+variables that describe media queries in your project. You can still use query strings for the first
+parameter.
```

Comment:
> You can still use query strings for the first parameter.

First parameter of what? How does this related to this interface?

Reply:
The `useMediaQuery` hook 🙂